tablesort.css

```diff
@@ -49,8 +49,4 @@ table .tablesort-sortable:hover:after, table .tablesort-asc:after, table .tables
 
 .showIfLast td {
     text-align: center;
-}
-
-.showIfLast td:after {
-    content: "No data";
```

Comment:
This is no longer needed since the text is now defaulted in the configuration which can be customized, and easily overridden on a per-table basis.